knowledge-base/drawer-chart-refresh.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Refresh a Chart Inside a Drawer
+description: How to refresh a Chart within a Drawer content.
+type: how-to
+page_title: Refresh a Chart Inside a Drawer
+slug: drawer-kb-chart-refresh
+position: 
+tags: 
+ticketid: 1583102
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Drawer for Blazor,<br /> Charts for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
+
+This knowledge base article answers the following questions:
+
+* How do I refresh a Chart inside a Drawer?
+* Is it possible to refresh a Chart when the Drawer is expanded?
+
+## Solution
+
+1. Create a bool flag and set it within the Drawer's [`ExpandedChanged`]({%slug drawer-events%}#expandedchanged) handler to indicate when the Chart needs to be refreshed
```

Comment:
Both list items (1. and 2.) need ending punctuation as per [our style guide](https://docs.telerik.com/style-guide/punctuation#lists).

As per our style guide, active voice is preferable over passive voice, especially if you're mixing the two within your sentence. Consider going with `to indicate when the Chart requires a refresh`.